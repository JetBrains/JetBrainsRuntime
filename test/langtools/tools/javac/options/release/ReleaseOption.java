/**
 * @test /nodynamiccopyright/
 * @bug 8072480
 * @summary Verify that javac rejects Java 17 program with --release 11
 * @compile ReleaseOption.java
 * @compile/fail/ref=ReleaseOption.out -XDrawDiagnostics --release 11 ReleaseOption.java
 */

interface ReleaseOption extends java.util.random.RandomGenerator {
}
