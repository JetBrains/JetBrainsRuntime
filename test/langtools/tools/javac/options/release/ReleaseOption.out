ReleaseOption.java:9:49: compiler.err.doesnt.exist: java.util.random
1 error
